xscen/data/fr/LC_MESSAGES/xscen.po

```diff
@@ -0,0 +1,35 @@
+# French translations for xscen.
+# Copyright (C) 2023 ORGANIZATION
+# This file is distributed under the same license as the xscen project.
+# Équipe Xscen <Rondeau-Genesse.Gabriel@ouranos.ca>, 2023.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: xscen 0.6.18b0\n"
+"Report-Msgid-Bugs-To: Rondeau-Genesse.Gabriel@ouranos.ca\n"
+"POT-Creation-Date: 2023-09-08 15:17-0400\n"
+"PO-Revision-Date: 2023-08-15 16:48-0400\n"
+"Last-Translator: Pascal Bourgault <bourgault.pascal@ouranos.ca>\n"
+"Language: fr\n"
+"Language-Team: fr <LL@li.org>\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.12.1\n"
+
+#: xscen/aggregate.py:184
+msgid "{window}-year mean of {attr}"
+msgstr "Moyenne {window} ans de {attr}"
+
+#: xscen/aggregate.py:318
+msgid "{attr1}: {kind} delta compared to {refhoriz}."
+msgstr "{attr1}: Delta {kind} comparé à {refhoriz}."
+
+#: xscen/diagnostics.py:500
+msgid "Ranking of measure performance"
+msgstr "Classement de performance de la mesure"
```

Comment:
Not sure of that one, but I don't have anything better...

Reply:
`Classement de la mesure de performance`??

Reply:
My word order fits my comprehension of what it is, but I agree that I was quite unsure on how to translate that...

@juliettelavoie  ?

Reply:
La traduction de pascal fonctionne. 

Dans la fonction, pour chaque propriété (et sa mesure associée), on donne un chiffre de 0 à 1 pour chaque réalisation. 0 = bon, genre un biais proche de 0 ou un ratio proche de 1